express/src/tracking/snowplow.ts

```diff
@@ -0,0 +1,10 @@
+import {
+  tracker,
+  gotEmitter,
+  HttpProtocol,
+  HttpMethod,
+} from "@snowplow/node-tracker";
+
+const emitter = gotEmitter("0.0.0.0", HttpProtocol.HTTP, 9090, HttpMethod.GET);
```

Comment:
We should rather use `POST` here as users might be inspired by the example and copy the setting.

Reply:
The micro collector does not work with HTTPS, that is why I added that 😆 
Do you think we should change it ?

Reply:
You can use Micro over HTTPS when using an [ngrok tunnel](https://ngrok.com/). I don't think it's that important here as people will use whatever they have. I think the protocol is a bit more important to show correctly as users might not have a preference/don't know the advantages of GET vs POST in our tracker implementation.

Reply:
Thank you, great explanation! Adding it now!